store/freelist/freelist.go

```diff
@@ -42,6 +44,13 @@ func OpenFreeList(path string) (*FreeList, error) {
 	}, nil
 }
 
+// SetOnUpdate sets a function to call when the freelist is updated.
+func (fl *FreeList) SetOnUpdate(f func()) {
```

Comment:
It is called when the primary gc is created, [here](https://github.com/ipld/go-storethehash/pull/64/files#diff-339abed331540529f949529ffe50e3f6527fd60420a76e6ed75ffce7118ff11fR43), and when it is closed, [here](https://github.com/ipld/go-storethehash/pull/64/files#diff-339abed331540529f949529ffe50e3f6527fd60420a76e6ed75ffce7118ff11fR63)

Probably did not see it because github collapsed the diff.